jans-cedarling/cedarling/src/init/policy_store.rs

```diff
@@ -19,6 +19,8 @@ pub enum LoadPolicyStoreError {
     MoreThanOnePolicy,
     #[error("could not found policy by id: {0}")]
     FindPolicy(String),
+    #[error("only one token should be assigned a `role_mapping`. The following tokens have a `role_mapping` field in your `policy_store.json`: {0}")]
+    MultipleRoleMappings(String),
 }
```

Comment:
In this case better to use MultipleRoleMappings(Vec<String>)

We want to return strong typed error when we can.


```suggestion
    #[error("only one token should be assigned a `role_mapping`. The following tokens have a `role_mapping` field in your `policy_store.json`: {0:?}")]
     MultipleRoleMappings(Vec<String>),
```

Reply:
Isn't it unnecessary in this case since Cedarling should panic and print/log the error when it receives a malformed config?  I don't think anyone should be using `match` on this error

Reply:
If user use rust, it always should return a result.  Error, it is also is result. Panicking is a bad practice. 

```
 I don't think anyone should be using match on this error
 ```
 But we should give possibility if we can.

Reply:
resolved here [782bcee](https://github.com/JanssenProject/jans/pull/9893/commits/782bceebf6159f73e8558f9ec5ee48638bb4b1c1)